webrecorder/browsertrix
# feat: Track collection events

Resolves https://github.com/webrecorder/browsertrix/issues/2224

## Changes

- Renames `inject_analytics` to `inject_extra` and updates docs
- Manually tracks page views to enable passing custom props
- Tracks copying collection share link and downloading a public collection

## Manual testing

1. Add `INJECT_EXTRA` to `.env.local` to match dev Helm chart value
2. Add `ANALYTICS_NAMESPACE=plausible` to your `.env.local`
3. Log in to org with public collections
4. Go to public collection
5. Click "Copy Link"
6. Log in to Plausible
7. Go to dev dashboard. Verify `[Collections] Copy share collection link` goal is logged
8. Paste in copied link
9. Go to Plausible dashboard. Verify `[Collections] View public collection` goal is logged
10. Go back to collection and click download
11. Go to Plausible dashboard. Verify `[Collections] Download public collection` goal is logged
